homeassistant/components/surepetcare/binary_sensor.py

```diff
@@ -8,7 +8,7 @@
 from homeassistant.components.binary_sensor import (
     DEVICE_CLASS_CONNECTIVITY,
     DEVICE_CLASS_PRESENCE,
-    BinarySensorEntity,
+    BinarySensorDevice,
```

Comment:
We've recently renamed the binary sensor entity base class. Please revert this change.

Reply:
Using BinarySensorEntity results in
`ERROR (MainThread) [homeassistant.setup] Unable to prepare setup for platform surepetcare.binary_sensor: Platform not found (cannot import name 'BinarySensorEntity' from 'homeassistant.components.binary_sensor' (/.../lib/python3.8/site-packages/homeassistant/components/binary_sensor/__init__.py)).`

🤔